3-module 5-task Найти min/max в строке Результат функция должна возвр…

ViMaCode opened the pull request:
Найти min/max в строке Результат функция должна возвращать в виде объекта

ViMaCode committed:
3-module 5-task Найти min/max в строке Результат функция должна возвращать в виде объекта

jsru commented:
Добавляю преподавателя (@SinimaWath) для код-ревью.

ViMaCode committed:
4-module 1-tasl # Функция makeFriendsList(friends)

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
4-module 2-task  # Выделите ячейки по диагонали

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
4-module 3-task # Функция highlight

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
5-module 1-task # Спрятать себя

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
5-module 2-task # Скрыть элемент по нажатию кнопки

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
5-module 3-task Карусель

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ViMaCode committed:
5-module 3-task Карусель (Замена моего решения)

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
6 модуль 1 задача start

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220113_v89111916010

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge branch 'master' of https://github.com/js-tasks-ru/jsbasic-20220113_v89111916010

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Учебный проект: Карусель 6 module 3 task

Создайте класс `Carousel`, описывающий компонент "Карусель".
С данным компонентом интерфейса мы уже познакомились
в предыдущем занятии, и для решения этой задачи,
вам понадобятся ваши наработки.
Главным отличием этой задачи будет использование компонентного подхода.

В качестве аргумента в конструктор класса
передаётся массив слайдов для отображения

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge branch 'js-tasks-ru:master' into master

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ViMaCode committed:
7-module 1-task

ViMaCode commented:
@

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode commented:
Hi There

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

ViMaCode committed:
7-module 2-task Модальное окно

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

ViMaCode committed:
7-module 3-task Пошаговый слайдер, часть 1

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

ViMaCode committed:
7-module 4-task Пошаговый слайдер, часть 2

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

ViMaCode committed:
8-module 1-task Учебный проект: Иконка корзины

Feat: Мое решение не проходит один тест

Возможен баг теста

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

ViMaCode committed:
Merge remote-tracking branch 'upstream/master'

jsru commented:
Решение было обновлено, посмотрим что скажет @SinimaWath
